moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -219,7 +220,7 @@ export default function TheMap({
       const isFeatureAlreadyInComponent = Boolean(
         currentComponent[tableToInsert].find(
           (feature) =>
-            feature?.[sourceFeatureId.toLowerCase()] === featureUniqueId || // Already in database
```

Comment:
I was relying on the fact that the database columns, `intersection_id` and `ctn_segment_id`, were lowercased versions of the names in the CTN metadata, `INTERSECTION_ID` and `CTN_SEGMENT_ID`. That is no longer the case with the new feature service and being more explicit is probably less brittle anyways.